test/include/test_util/fs_util.h

```diff
@@ -5,19 +5,20 @@
 
 namespace noisepage::common {
 /**
- * Determine the absolute path to the project root directory (e.g. noisepage/).
- * @return The absolute path to the project root, as a string.
+ * Determine the absolute path to the build directory for the current build (e.g. noisepage/build/)
+ * @return The absolute path to the build directory, as a string.
  */
-std::string GetProjectRootPath();
+std::string GetBuildRootPath();
 
 /**
- * Determine the absolute path to the file with the given name by performing a
- * recursive directory search from the specified root path.
- * @param filename The name of the target file
- * @param root_path The absolute path from which recursive search should begin
- * @return The absolute path to a file with the specified name, as a string.
- * @throw std::runtime_error in the event the file cannot be found.
+ * Determine the absolute path to the build artifact identified by `name`.
+ *
+ * By using this function, you are expressing the assumption that the
+ * artifact identified by `name` is located in the binary directory for
+ * the current build; that is, `name` is located at `build/bin/<name>`.
```

Comment:
It doesn't have to be `build` right? Just whatever the build root is? `GetBuildRootPath()/bin/name` might be clearer, if so (because we do hardcode some assumptions about `build` elsewhere)

Reply:
Thanks for the comment; you are right, the whole point of the function is to avoid backing-in logic that relies on a specific directory name / path. Updated it!